process/transaction/shardProcess.go

```diff
@@ -800,12 +790,15 @@ func (txProc *txProcessor) executeFailedRelayedTransaction(
 		return err
 	}
 
+	//TODO: check if the totalFee can be smaller, in case of a moveBalance for example
```

Comment:
not possible.

Reply:
ok

Reply:
I think right now it will happend, because the GasLimit is multiplied by the gasModifier here. So it is divided by 100.

Reply:
totalFee cannot be lower than moveBalanceFee. it is always moveBalance + extra fee.

Reply:
ok not here, but in the next if